docs/content/docs/connectors/table/hive/overview.md

```diff
@@ -271,6 +271,13 @@ Please find the required dependencies for different Hive major versions below.
 {{< /tab >}}
 {{< /tabs >}}
 
+#### Moving the planner jar
+
+Move the jar `flink-table-planner_2.12` located in `FLINK_HOME/opt` to `FLINK_HOME/lib` and move out the jar `flink-table-planner-loader` from `FLINK_HOME/lib`.
```

Comment:
We can use the full jar name here by using `{{< version >}}` variable. Would be better to provide several commands (`mv xxx`) to complete the work.